app/views/dashboards/current_tasks/_task.html.erb

```diff
@@ -0,0 +1,15 @@
+<a class="current-task-item" href="<%= protocols_my_module_path(task.id) %>">
+  <div class="current-task-breadcrumbs"><%= task.experiment.project.name %>
+    <span class="slash">/</span><%= task.experiment.name %></div>
+  <div class="task-name row-border">
+    <%= task.name %>
+  </div>
+  <div class="task-due-date row-border <%= prepare_due_date(task)[:state ]%>">
+                                  <span class="<%= prepare_due_date(task)[:text] ? '' : 'hidden' %>">
```

Comment:
why text here ?

Reply:
Because the `prepare_due_date` method is returning us formatted text. Its some logic behind this method

Reply:
ufff, nice catch! thanks!

Reply:
Copy-paste from the previous template. Check. comment above..

Reply:
Check comment below

Reply:
Ok